apps/remix-ide/src/app/panels/file-panel.js

```diff
@@ -57,7 +57,7 @@ module.exports = class Filepanel extends ViewPlugin {
     this.slitherHandle = new SlitherHandle()
     this.workspaces = []
     this.appManager = appManager
-    this.currentWorkspaceMetadata = {}
+    this.currentWorkspaceMetadata = null
```

Comment:
Is this change needed?

Reply:
fartther you check if if it is null but it is an empty object